### DISCUSSION TITLE
xunit.runner.console for .NET 8.0

### DISCUSSION THREAD

**Discussion by AdaskoTheBeAsT** [xunit/xunit#2825] [Question] [answered]
Hello,

will it be possible to compile xunit.runner.console also for .NET 8? I know that it is not supporter - I have advanced scenario when I am running xunit under dotcover - I migrate large project from .net 4.8.1 to net .8.0 - by multitarget libs against both frameworks - for testing coverage I use dotcover where xunit is run (build pipeline defined in Cake)
when having only .net 8 installed in azure agent this is what we see

![image](https://github.com/xunit/xunit/assets/1914201/66569bab-212d-4565-8635-624f3c510369)

as i checked xunit.runner.console is targeted only at max .net 6

when I installed also .net 6 on agent I receive this error

![image](https://github.com/xunit/xunit/assets/1914201/f267a2e8-e6a9-404f-bd74-4f3d68d2cd6c)

if you only can release console with additional .NET 8 as target framework this would fix all my team headaches

I can prepare PR for this

**Comment by bradwilson** [comment, marked as answer]
The .NET Core runners in `xunit.runner.console` are not supported. They exist solely for a team at Microsoft.

Everybody else is expected to use `xunit.runner.visualstudio` and `dotnet test` for .NET Core/.NET 5+.
